front/app/components/FormBuilder/components/FormBuilderSettings/ContentSettings/index.tsx

```diff
@@ -61,8 +60,8 @@ export const ContentSettings = ({
       <Box mt="16px">
         {!isFieldGrouping && (
           <>
-            {!isNilOrError(field.isTitleEditable)
-              ? field.isTitleEditable
+            {!isNilOrError(lockedAttributes?.title_multiloc)
```

Comment:
Since we are trying to ascertain a boolean, this can be simplified to something like

```
            {!lockedAttributes?.title_multiloc && (
                  <SectionField>
                    <InputMultilocWithLocaleSwitcher
                      initiallySelectedLocale={platformLocale}
                      id="e2e-title-multiloc"
                      name={`customFields.${field.index}.title_multiloc`}
                      label={<FormattedMessage {...messages.questionTitle} />}
                      type="text"
                    />
                  </SectionField>
                )}
```